src/refactor_nrepl/find/find_macros.clj

```diff
@@ -61,14 +66,14 @@
               (recur macros (reader/read opts rdr)))))))))
 
 (defn- get-cached-macro-definitions [f]
-  (when-let [[ts v] (get @cache (.getAbsolutePath f))]
+  (when-let [[ts v] (get @macro-defs-cache (.getAbsolutePath f))]
```

Comment:
ooooooooooooh my bad!  I read the diff wrong, I thought you changed the atom, but you renamed and added another one to suit your use-case.